bin/c8.js

```diff
@@ -27,8 +27,9 @@ let argv = buildYargs().parse(instrumenterArgs)
     }
 
     process.env.NODE_V8_COVERAGE = argv.tempDirectory
-    foreground(hideInstrumenterArgs(argv), () => {
+    foreground(hideInstrumenterArgs(argv), done => {
       outputReport(argv)
+      done()
```

Comment:
Not needed. Even If no callback function is provided, foreground-child itself will do what you do here. https://github.com/tapjs/foreground-child/blob/v1.5.6/index.js#L22-L24

Reply:
But c8 has pass ths cb argument without done.

Reply:
Indeed.